Helpwidget update

chrisgarrity opened the pull request:
### Resolves:

Addressing followup comments from: https://github.com/LLK/scratch-www/pull/3800 and meeting discussion.

### Changes:
* delays loading the freshdesk script. _Note: it does not maintain a reference because there are two instances of the widget on the page. If the script gets loaded twice, the launcher button for one of them always appears._
* wording changes to body copy
* moves the profile or studio link out of the description into a new custom field (will now require the sender to fill in something in the description to say why).

### Test Coverage:
* contact us on staging:
  * without feature-flag should show current page with the form
  * with feature-flag `CONTACT_US_POPUP=true` shows new copy and opens widget when the Get Help link or button is clicked.
* if linked to from the inappropriate profile or studio contact us the link will be prefilled into the new link field not the description

apple502j commented on src/views/contact-us/contact-us.jsx:
```diff
@@ -8,7 +8,7 @@ const render = require('../../lib/render.jsx');
 const HelpForm = require('../../components/helpform/helpform.jsx');
 const HelpWidget = require('../../components/helpwidget/helpwidget.jsx');
-const {CONTACT_US_POPUP} = require('../../lib/feature-flags.js');
+const {CONTACT_US_POPUP} = true;
```
```suggestion
const CONTACT_US_POPUP = true;
```

It will be `undefined` otherwise because `true` does not have property `CONTACT_US_POPUP`

chrisgarrity commented:
@rschamp the helpwidget PR is failing the test for duplicate strings because I added the helpwidget 'Send' string to the general l10n file, and there is already a report 'Send' string. I think the right thing to do now is remove the test, not the string. The test was added because duplicate strings broke the conversion to/from PO files before we switched www (https://github.com/LLK/scratch-www/issues/1102). That's no longer relevant. /cc @BryceLTaylor
